schemas/context/profileStitch.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profileStitch",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile Stitch",
+  "type": "object",
+  "description": "Details about the ids that were joined by profile stitching.",
```

Comment:
This seems like a very Adobe-centric thing, yes? If so, it belongs in an extension and not in the primary schema

Reply:
Please see earlier comments on this. I am confident this is not an Adobe-centric things at all.